classes/Kohana/HTTP.php

```diff
@@ -203,16 +211,12 @@ public static function request_headers()
 	public static function www_form_urlencode(array $params = array())
 	{
 		if ( ! $params)
-			return;
-
-		$encoded = array();
+			return '';
```

Comment:
This changes the API of the function. This could cause unexpected behaviour of people that are using it.